src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/Form.js

```diff
@@ -1,58 +1,39 @@
 // @flow
 import {observer} from 'mobx-react';
-import type {ElementRef} from 'react';
 import React from 'react';
 import Loader from '../../components/Loader';
 import Renderer from './Renderer';
 import FormStore from './stores/FormStore';
-import formStyles from './form.scss';
 
 type Props = {
     store: FormStore,
-    onSubmit: () => void,
+    onSubmit: (action: string) => void,
```

Comment:
Why don't we allow `undefined` here as well? Might be awkward in some situation to think about an action name.